src/Surfnet/StepupGateway/SecondFactorOnlyBundle/Controller/SecondFactorOnlyController.php

```diff
@@ -140,8 +140,7 @@ public function respondAction()
                 [
                     'acu' => $responseContext->getDestinationForAdfs(),
                     'samlResponse' => $xmlResponse,
-                    'context' => $adfsParameters->getContext(),
-                    'authMethod' => $adfsParameters->getAuthMethod(),
+                    'adfs' => $adfsParameters,
```

Comment:
To use the adfs partial, the existing SFO controller action that handled the adfs response, no sends the adfs parameters value object directly to the view.  The view tests if the parameters are present, and adds them to the post params.